jenkins-pipelines/nemesis/longevity-5gb-1h-PauseLdapNemesis-aws.jenkinsfile

```diff
@@ -7,6 +7,6 @@ longevityPipeline(
     backend: 'aws',
     region: 'eu-west-1',
     test_name: 'longevity_test.LongevityTest.test_custom_time',
-    test_config: '''["test-cases/nemesis/longevity-5gb-1h-PauseLdapNemesis.yaml", "configurations/ldap-authorization.yaml"]'''
```

Comment:
same as above, ldap need to be adapted correctly as well

Reply:
Fixed